web_src/js/features/repo-projects.js

```diff
@@ -183,26 +183,19 @@ export function initRepoProject() {
 }
 
 function setLabelColor(label, color) {
-  const red = getRelativeColor(parseInt(color.slice(1, 3), 16));
-  const green = getRelativeColor(parseInt(color.slice(3, 5), 16));
-  const blue = getRelativeColor(parseInt(color.slice(5, 7), 16));
-  const luminance = 0.2126 * red + 0.7152 * green + 0.0722 * blue;
+  // sRGB color space luminance
+  const r = parseInt(color.slice(1, 3), 16);
+  const g = parseInt(color.slice(3, 5), 16);
+  const b = parseInt(color.slice(5, 7), 16);
+  const luminance = (0.299 * r + 0.587 * g + 0.114 * b) / 255;
 
-  if (luminance > 0.179) {
-    label.removeClass('light-label').addClass('dark-label');
-  } else {
+  if (luminance < 0.35) {
```

Comment:
https://stackoverflow.com/a/596243

If the new formula has been tested and it's actually better, a comment is enough.

Reply:
Probably the best solution would to use APCA in WCAG 3.0, though that is not finished yet and I could not find a simple implementation to use. And it would have to be duplicated in Go and Javascript.

All of these formulas are basically made up based on what looks good empirically. The standard Rec.709 luminance as used before has no particular reason for being the right choice for contrast when it comes to human perception.

I will add a comment.